localcache.go

```diff
@@ -0,0 +1,22 @@
+package localcache
+
+import (
+	"errors"
+	"time"
+)
+
+var (
+	defaultTTL = time.Second * 30
+	// ErrKeyNotExist means that key doesn't exist.
+	ErrKeyNotExist = errors.New("key does not exist.")
```

Comment:
A small suggestion: move these two Error messages from impl.go to localcache.go. It's a public error need to be known outside.